### DISCUSSION TITLE
Combining capacitor plugins (Camera Preview and Barcode Scanner)

### DISCUSSION THREAD

**Discussion by jasonworkboost** [ionic-team/capacitor#3344] [Q&A]
Dear all,

I have started working with capacitor and I love it! I was able to make a simple QR code scanner using this tutorial:
https://enappd.com/blog/qr-code-and-barcode-scanning-with-ionic-capacitor/127/

Now I would like to extend this by showing information on the topscreen and have the scanner on the bottom of the screen. With camera preview I am able to do this: https://github.com/capacitor-community/camera-preview

However, how I am able to combine them? I have been googling quite some time, but I can't get it to work. I am fairly new to this, so my appologies if I am asking a silly question. Any pointers to where to look or some tips would be really helpfull. Thank you in advance.

**Comment by Juphex** [comment]
I am on the same question and I now try to find out how to stream Capacitors data. Have you any progress yet=

**Reply by jasedu** [reply]
Nope sorry, we eventually settled for just the camera and no overlay. If you find a fitting solution, please let me know!

**Reply by Juphex** [reply]
Using Capacitor? How did you do that, are there params that allow to remove the overlay?

**Reply by jasedu** [reply]
not to remove the overlay, but to remove the buttons etc

**Comment by UltimateGemini** [comment]
Anyone got this working yet?

**Reply by jasonworkboost** [reply]
nope sorry, still no solution for this...
